catroid/src/main/res/values/strings.xml

```diff
@@ -1337,44 +1330,44 @@
     <!--  -->
 
     <!-- Error messages -->
-    <string name="error_no_writiable_external_storage_available">This app needs writable external storage like a SD
-        card in order to run, but none was found.\nYou can fix this by making the external storage writable or inserting
-        a SD card, then trying again.</string>
-    <string name="error_no_write_access">This app needs permission to write data to your device. Otherwise you
-        can\'t create, modify or run programs. To continue, open settings and enable write access for
-        this app.</string>
-    <string name="error_project_upload">An error occurred while uploading the program.</string>
-    <string name="error_project_download">An error occurred while downloading the program.</string>
-    <string name="error_new_project">An error occurred while creating a new program.</string>
+    <string name="error_no_writable_external_storage_available">This app needs writable external storage like an
```

Comment:
```suggestion
    <string name="error_no_writable_external_storage_available">This app needs writable external storage like an
```
This typo correction will most likely be done in https://jira.catrob.at/browse/CATROID-57 so it may or may not be done here.